python-packages/input4MIPs-CVs/src/input4MIPs_CVs/html_generation.py

```diff
@@ -881,10 +912,13 @@ def write_db_view_as_html(
         f"<p><h1>{page_title}: v{version}</h1><p>",
         "<h4>",
         "<a href='/'>Home</a>",
-        "| <a href='input4MIPs_delivery-summary_CMIP6Plus.html'>Delivery summary view</a>",
-        "| <a href='input4MIPs_source-id_CMIP6Plus.html'>Source ID-level view</a>",
-        "| <a href='input4MIPs_datasets_CMIP6Plus.html'>Dataset-level view</a>",
-        "| <a href='input4MIPs_files_CMIP6Plus.html'>File-level view</a>",
+        "| <a href='input4MIPs_delivery-summary.html'>Delivery summary view</a>",
+        "| <a href='input4MIPs_source-id_CMIP7.html'>Source ID-level view</a>",
```

Comment:
makes sense to identify this by mip_era = CMIP7, the CMIP6Plus is explicit - finally catching the template code that is written into the HTML files